cluster/addons/fluentd-elasticsearch/es-pod.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1beta1
+kind: Pod
```

Comment:
These need to run under replication controllers, not bare pods.  Monitoring is wrong here, too @vishh #2408